pkgs/top-level/perl-packages.nix

```diff
@@ -157,7 +157,7 @@ let
       url = "mirror://cpan/authors/id/J/JF/JFREEMAN/Algorithm-LCSS-0.01.tar.gz";
       sha256 = "0y0zc3sq283zpv67vy7a3h3dyvjn5svjxwknanmp38a2g36fyz3i";
     };
-    propagatedBuildInputs = [ AlgorithmDiff ];
```

Comment:
this may be wrong, it is one of the packages you recently added, and cpan2nix has no glue. it took deps from cpan metadata, which is often invalid